Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Tabular/DriverInformation.hs

```diff
@@ -52,6 +52,8 @@ mkPersist
       canDowngradeToTaxi Bool
       mode Domain.DriverMode Maybe
       aadhaarVerified Bool
+      blockedReason Text Maybe
```

Comment:
is this supposed to be a foreign key to `driver_block_reason` table ?

Reply:
no

Reply:
what is this value then ? also we should have some link between these two tables.

Reply:
okk but i thought  from this we can see what is the reason  for which driver is blocked

Reply:
why we need to relate these two tables

Reply:
then it's a foreign key to that table right ? the value of `blockedReason` here should exist in the table `driver_block_reason` as a `reasonCode` right ?

Reply:
lets the reason_code is others so  this value will not be fixed driver_block_reason

Reply:
we can have two fields, 
one `blocking_reason_code: Maybe (ReasonCodeId)` and `blocking_comment: Maybe (Text)`.

Reply:
there is no such rule that fk can't be null, the only rule is if a referencing table contains a value then there should be an entry for that value (key) in the referenced table. It depends on the use case if we want to make it `NULLABLE` or `NOT NULL` in the referencing table.

Reply:
ok